src/Message/AIMAuthorizeRequest.php

```diff
@@ -46,6 +48,17 @@ protected function addPayment(\SimpleXMLElement $data)
         }
     }
 
+    protected function addDescription(\SimpleXMLElement $data)
```

Comment:
Before the change, description was added within addPayment method.

AuthorizeNET API XSD schema requires SolutionID to be added in between Payment and Order->Description tags, so I split addPayment into two methods: addPayment and addDescription so they can be called independently.